person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   atrr_reader :id, :name, :age
   atrr_writer :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] I couldn't see any method since you inherited namable class so also you should implement it here as well and it should return the name so kindly implement it 💯✨

![image](https://user-images.githubusercontent.com/58553711/210457111-498f1b73-20ef-4f98-a4dd-d14cad2df3c0.png)

Reply:
Thank you